js/modules/tamagotchi.js

```diff
@@ -9,10 +9,51 @@ export default class Tamagotchi {
 
   displayHealth = (elementSelector) => {
     const displayElement = document.querySelector(elementSelector);
-    displayElement.innerText = this.energy.value;
+    displayElement.innerText = this.health.value;
   };
 
+  decreaseParameters = () => {
+
+    const healthInterval = setInterval(() => {
```

Comment:
**nitpick:** We should not create such big functions. We should split in into smaller pieces. Each of of parameter decrease should be a separate method. Not it's impossible to re-use these methods at any point